.github/workflows/build-docker.yml

```diff
@@ -176,13 +176,13 @@ jobs:
             ${{ github.workspace }}/HPCC-Platform/.github/workflows/timeoutcmd
           if-no-files-found: error
 
-      - name: Upload UI Test Files
+      - name: Upload ECL Watch UI Test Files
         if: ${{ inputs.upload-package == true }}
         uses: actions/upload-artifact@v3
         with:
-          name: ${{ inputs.asset-name }}-ui_test-files
+          name: ${{ inputs.asset-name }}-ecl_watch_ui_tests
           path: |
-            ${{ github.workspace }}/HPCC-Platform/esp/src/test-ui/**/*
+            ${{ github.workspace }}/HPCC-Platform/esp/src/test-ui/tests/framework/**/*
```

Comment:
here is why that test is failing. There is nothing under tests/framework in this PR. Perhaps he's adding it in at a later time and we can just ignore this for now?